website/src/docs/dragdrop.md

```diff
@@ -27,6 +26,8 @@ uppy.use(DragDrop, {
 })
 ```
 
+> Note that certain [restrictions set in Uppy’s main options](/docs/uppy#restrictions), namely `maxNumberOfFiles` and `allowedFileTypes`, affect the system file picker dialog. If `maxNumberOfFiles: 1`, users will only be able to select one file, and `allowedFileTypes: ['image/*, .jpg']` means only image or files with `.jpg` extension will be selectable.
```

Comment:
It might be clearer to have non overlapping example here, eg "`allowedFileTypes: ['image/*', '.pdf']` means only images and PDFs will be selectable"

Reply:
I like it, thanks! ❤️